docs/addons/fluxcd.md

```diff
@@ -13,22 +13,62 @@ import * as blueprints from '@aws-quickstart/eks-blueprints';
 const app = new cdk.App();
 
 const addOn = new blueprints.addons.FluxCDAddOn({
-  bootstrapRepo: {
-      repoUrl: 'https://github.com/stefanprodan/podinfo',
-      name: "podinfo",
-      targetRevision: "master",
-      path: "./kustomize"
-  },
-  bootstrapValues: {
-      "region": "us-east-1"
-  },
+    bootstrapRepo: {
+        repoUrl: 'https://github.com/stefanprodan/podinfo',
+        name: "podinfo",
+        targetRevision: "master",
+        path: "./kustomize"
+    },
+    bootstrapValues: {
+        "region": "us-east-1"
+    },
 }),
 
 const blueprint = blueprints.EksBlueprint.builder()
-  .addOns(addOn)
-  .build(app, 'my-stack-name');
+    .addOns(addOn)
+    .build(app, 'my-stack-name');
+```
+
+## Workload Repositories
+
+1. To add workload repositories as well as the bootstrap repository, please follow this example below 
+
+```typescript
+import * as cdk from 'aws-cdk-lib';
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+
+const app = new cdk.App();
+
+const addOn = new blueprints.addons.FluxCDAddOn({
+    bootstrapRepo: {
+        repoUrl: 'https://github.com/stefanprodan/podinfo',
+        name: "podinfo",
+        targetRevision: "master",
+        path: "./kustomize"
+    },
+    bootstrapValues: {
+        "region": "us-east-1"
+    },
+    workloadApplications: [
+        {
+            name: "nginx",
+            namespace: "default",
+            repository: {
+                repoUrl: 'https://github.com/zjaco13/flux-tester',

```

Comment:
Could we use a standard repos vs pointing to your own personal repo? Something like this - https://github.com/aws-observability/aws-observability-accelerator/tree/main or PodInfo

Reply:
We can use another repo.  Not sure if the observability accelerator manifests will be the best option though, because they require values and a different namespace.